portfolio/src/main/webapp/script.js

```diff
@@ -159,10 +159,10 @@ function drawChart() {
   const data = new google.visualization.DataTable();
   data.addColumn('string', 'Person');
   data.addColumn('number', 'Percentage');
-        data.addRows([
-          ['Men', 72],
-          ['Women', 28],
-        ]);
+  data.addRows([
+  ['Men', 72],
+  ['Women', 28],
```

Comment:
Nit: in line 163-164, I think you may still want to use some indentation here. I believe it's 4-space indentation if you break a statement.